jetbrains-core/tst/software/aws/toolkits/jetbrains/services/s3/DeleteBucketTest.kt

```diff
@@ -126,7 +126,7 @@ class DeleteBucketTest {
             assertThat(openEditor(projectRule.project, bucket)).isNotNull
```

Comment:
Should probably add to the tests by having a test where the client throws `BucketNotExist` exceptions

Reply:
Added a test